package-support/init.d/sideboard

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
```

Comment:
MAGFest won't use it anytime soon, but we have an ``init.d`` script now for Sideboard (we currently use ``supervisord`` instead, which works better for the way we deploy).